week4.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>is "grainsize" hyphenated or one-word?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I see it all different ways... one word, two words, or hyphenated. I'll go through and just try to make sure I'm being consistent everywhere - thanks!